### ISSUE TITLE
Optionally allow attributes and targets on the same line

### ISSUE THREAD

**Issue by stlrnz** [slevomat/coding-standard#1480]
I've tested the `SlevomatCodingStandard.Attributes.AttributeAndTargetSpacing` Sniff and stumbled upon an edge case.
While I agree that an attributes and its target should not be on the same line, there is one situation where I'd like to allow it.

Symfony introduced in 5.2 the `#[CurrentUser]` attribute for controller arguments: https://symfony.com/blog/new-in-symfony-5-2-controller-argument-attributes

```php
<?php

class IndexController
{
    public function index(
        #[CurrentUser] ?User $user,
        SomeServce $someServce,
        SomeOtherService $someOtherService
    ): Response {
        //do something
    }
}
```

It would be great if either
a) the error followed by `$areOnSameLine` could be disabled (e.g. by the new property `allowOnSameLine`)
https://github.com/slevomat/coding-standard/blob/da14851de59f39fb68556cb4e5cbf0a68ca27276/SlevomatCodingStandard/Sniffs/Attributes/AttributeAndTargetSpacingSniff.php#L60

b) or by a special handling for argument attributes (controlled by the new property `allowArgumentAttributesOnSameLine`).

What do you think?

**Comment by kukulich** [comment]
I'm ok with `allowOnSameLine` option.
